From b016dcd2f76a95350ae22ae24ffbef567ec62012 Mon Sep 17 00:00:00 2001
From: Ulf Hermann <ulf.hermann@qt.io>
Date: Wed, 15 Oct 2025 08:28:27 +0200
Subject: [PATCH 144/239] Skip
 tst_QQuickColorDialogImpl::dialogCanMoveBetweenWindows()

This crashes with a qFatal(), so no blacklisting will help. It only
happens on 6.10. Therefore we're not picking back from dev.

Task-number: QTBUG-141162
Change-Id: I62a5997bad47f6094f0263c08ada212198a28ee4
Reviewed-by: Santhosh Kumar <santhosh.kumar.selvaraj@qt.io>
Reviewed-by: Oliver Eftevaag <oliver.eftevaag@qt.io>
---
 .../qquickcolordialogimpl/tst_qquickcolordialogimpl.cpp      | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/auto/quickdialogs/qquickcolordialogimpl/tst_qquickcolordialogimpl.cpp b/tests/auto/quickdialogs/qquickcolordialogimpl/tst_qquickcolordialogimpl.cpp
index 8c4bc9693b..f4bebd06ac 100644
--- a/tests/auto/quickdialogs/qquickcolordialogimpl/tst_qquickcolordialogimpl.cpp
+++ b/tests/auto/quickdialogs/qquickcolordialogimpl/tst_qquickcolordialogimpl.cpp
@@ -629,6 +629,11 @@ void tst_QQuickColorDialogImpl::workingInsideQQuickViewer() // QTBUG-106598
 
 void tst_QQuickColorDialogImpl::dialogCanMoveBetweenWindows()
 {
+#ifdef Q_OS_WINDOWS
+    // QTBUG-141162
+    QSKIP("Crashes on windows");
+#endif
+
     DialogTestHelper<QQuickColorDialog, QQuickColorDialogImpl> dialogHelper(this, "windowSwapping.qml");
     QVERIFY2(dialogHelper.isWindowInitialized(), dialogHelper.failureMessage());
     QVERIFY(dialogHelper.waitForWindowActive());
-- 
2.51.2

